Source/cmake/WebKitCompilerFlags.cmake

```diff
@@ -253,12 +253,18 @@ if ("${CMAKE_CXX_COMPILER_ID}" STREQUAL "GNU" AND NOT "${LOWERCASE_CMAKE_HOST_SY
     set(CMAKE_SHARED_LINKER_FLAGS_DEBUG "-Wl,--no-keep-memory ${CMAKE_SHARED_LINKER_FLAGS_DEBUG}")
 endif ()
 
-if (LTO_MODE AND COMPILER_IS_CLANG)
+if (LTO_MODE AND COMPILER_IS_CLANG AND NOT MSVC)
     set(CMAKE_C_FLAGS "-flto=${LTO_MODE} ${CMAKE_C_FLAGS}")
     set(CMAKE_CXX_FLAGS "-flto=${LTO_MODE} ${CMAKE_CXX_FLAGS}")
     set(CMAKE_EXE_LINKER_FLAGS "-flto=${LTO_MODE} ${CMAKE_EXE_LINKER_FLAGS}")
     set(CMAKE_SHARED_LINKER_FLAGS "-flto=${LTO_MODE} ${CMAKE_SHARED_LINKER_FLAGS}")
     set(CMAKE_MODULE_LINKER_FLAGS "-flto=${LTO_MODE} ${CMAKE_MODULE_LINKER_FLAGS}")
+elseif (LTO_MODE AND COMPILER_IS_CLANG AND MSVC AND NOT DEVELOPER_MODE)
```

Comment:
What's the reason you add `NOT DEVELOPER_MODE`?

Reply:
There's a couple of link errors for WebCore with LTO in developer mode:

```
lld-link: error: undefined symbol: __declspec(dllimport) currentStackPointer
>>> referenced by S:\WebKit\WebKitBuild\Debug\WebCore\DerivedSources\unified-sources\UnifiedSource-68aea4ac-21.cpp
>>>               Source\WebCore\CMakeFiles\WebCore.dir\__\__\WebCore\DerivedSources\unified-sources\UnifiedSource-68aea4ac-21.cpp.obj

lld-link: error: undefined symbol: private: static class WebCore::BreakLines::LineBreakTable const WebCore::BreakLines::lineBreakTable
>>> referenced by S:\WebKit\WebKitBuild\Debug\WebCore\DerivedSources\unified-sources\UnifiedSource-207b877e-6.cpp
>>>               Source\WebCore\CMakeFiles\WebCore.dir\__\__\WebCore\DerivedSources\unified-sources\UnifiedSource-207b877e-6.cpp.obj
>>> referenced by S:\WebKit\WebKitBuild\Debug\WebCore\DerivedSources\unified-sources\UnifiedSource-043dd90b-15.cpp
>>>               Source\WebCore\CMakeFiles\WebCore.dir\__\__\WebCore\DerivedSources\unified-sources\UnifiedSource-043dd90b-15.cpp.obj
>>> referenced by S:\WebKit\WebKitBuild\Debug\WebCore\DerivedSources\unified-sources\UnifiedSource-043dd90b-18.cpp
>>>               Source\WebCore\CMakeFiles\WebCore.dir\__\__\WebCore\DerivedSources\unified-sources\UnifiedSource-043dd90b-18.cpp.obj
ninja: build stopped: subcommand failed.
```

The first is something across a DLL boundary but the second I'm not sure.